scripts/common.sh

```diff
@@ -46,15 +49,17 @@ phase() {
 
 # Inspect the environment variables to assert the minimum configuration is informed.
 assert_required_configuration_or_fail() {
-    [[ -z "${PARAMS_URL}" ]] &&
-        fail "Parameter URL is not set!"
+    ([[ -z "${PARAMS_URL}" ]] && [[ -z "${PARAMS_GIT_SCRIPT}" ]]) &&
+        fail "Parameter URL or SCRIPT is not set!"
```

Comment:
Good move 👍 I think the erro message should mention something like "Only Param URL or SCRIPT must be set!". And also, please consider the following syntax:

```bash
    [[ -z "${PARAMS_URL}" -a -z "${PARAMS_GIT_SCRIPT}" ]] &&
        fail "Only Param URL or SCRIPT must be set!"
```

Reply:
instead of -a, i had to &&